src/database/medicine/medicine.repository.ts

```diff
@@ -0,0 +1,64 @@
+import { MedicineModel } from "../../modules/medicine/models/medicine.interface";
+import { MedicineDbModel } from "./medicineDb.interface";
+import { getDb } from '../connection';
+import { config } from "../../config";
+
+export class MedicineRepository {
+    constructor(){}
+
+    //Could just do this manually not sure we need to add from the web app
```

Comment:
yup, no need to add another UI component for this, we'll just populate with postman